Scripts/Setup/setupExperiment.sh

```diff
@@ -227,7 +228,7 @@ if [[ "${DATATYPE}" == 'CMIP5' ]]; then
   POPMAP=${POPMAP:-'map_gx1v6_to_fv0.9x1.25_aave_da_090309.nc'} # ocean grid definition
   METGRIDTBL=${METGRIDTBL:-'METGRID.TBL.CESM'}
 elif [[ "${DATATYPE}" == 'CMIP6' ]]; then
-  VTABLE=${VTABLE:-'Vtable.CMIP6.'${CMIP6MODEL}'.csv'}
+  VTABLE=${VTABLE:-'Vtable.CMIP6.'${CMIP6MODEL}'_'${CMIP6MDLVER}'.csv'}
```

Comment:
The main issue I see here is that this forces the use of a version for every model (and PROJ is not a good default either) - you can either make it options (with an if-clause) or just get rid of it. I think we determined that this is the only case where CMIP6MODEL is used, so we don't really need two variables here.

Reply:
Fixed.